themes/theme-default/package.json

```diff
@@ -56,6 +56,9 @@
     "vue": "^3.4.15",
     "vue-router": "^4.2.5"
   },
+  "devDependencies": {
+    "vitest": "^1.1.0"
+  },
```

Comment:
I am not sure, but I think declaring vitest at workspace root should be enough? Is there any necessary to declare here?

Reply:
Removed.